pkg/k8s/apis/cilium.io/v2/client/register.go

```diff
@@ -95,13 +95,18 @@ func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) er
 		return createIdentityCRD(clientset)
 	})
 
+	// Reviewers: I'm not sure if it makes sense to wrap this under the feature
+	// flag check since we have related Makefile changes.
```

Comment:
At a glance, I would assume we should probably not register the CRD if the feature is disabled.

Reply:
Contrary opinion, discuss there: https://github.com/cilium/cilium/pull/13671#discussion_r509806503